src/components/Calculator.jsx

```diff
@@ -0,0 +1,145 @@
+      </p>
+      <div class="grid grid-rows-3 grid-cols-2 grid-flow-row gap-8">
+        <Slider
+          min="1"
+          max="100"
+          value={developersCount}
+          description={`How many developers are on the project?`}
+          setValue={setDevelopersCount}
+          name={(value) => `${value} developers`}
+          id="developers"
+        />
+        <Slider
+          min="1"
+          max="60"
+          value={averageBuildTime}
+          description={`How long does the project take (minutes) to build?`}
+          setValue={setAverageBuildTime}
+          name={(value) => `${value} minutes`}
+          id="build-time"
+        />
+        <Slider
+          min="1"
+          max="100"
+          value={buildTimeSaved}
+          description={`In average in every build.`}
+          setValue={setBuildTimeSaved}
+          name={(value) => `${value}% of time saved`}
+          id="build-time-saved"
+        />
+        <Slider
+          min="1000"
+          max="20000"
+          value={averageMonthlySalary}
+          description={`What is the cost to the company?`}
+          setValue={setAverageMonthlySalary}
+          name={(value) => `Monthly salary of $${value}`}
+          id="salary"
+        />
+        <Slider
+          min="1"
+          max="100"
+          value={ciBuildCost}
+          description={`On CI (e.g. GitHub Actions, Bitrise...)`}
+          setValue={setCIBuildCost}
+          name={(value) => `$${value / 100} per build minute`}
+          id="ci-build-cost"
+        />
+        <Slider
+          min="1"
+          max="50"
+          value={localBuilds}
+          description={`Per developer and day.`}
+          setValue={setLocalBuilds}
+          name={(value) => `${value} local clean builds`}
+          id="local-builds"
+        />
+        <Slider
+          min="1"
+          max="50"
+          value={ciBuilds}
+          description={`Per developer and day.`}
+          setValue={setCIBuilds}
+          name={(value) => `${value} CI clean builds`}
+          id="ci-builds"
+        />
+      </div>
+      <div class="grid grid-cols-2 mt-10">
+        <div class="flex flex-col gap-y-3">
+            <div class="uppercase text-slate-500 text-xs">
+            Annual Developer Hours Saved
+            </div>
+            <div class="text-white text-2xl font-bold">
+            {addDots(timeSaved)} hours/year
+            </div>
+        </div>
+
+        <div class="flex flex-col gap-y-3">
+            <div class="uppercase text-slate-500 text-xs">
+            Annual R&D Cost Recapture
+            </div>
+            <div class="text-white text-2xl font-bold">
+            ${addDots(moneySaved)}
+            </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function addDots(number) {
+    return number.toString().replace(/(\d)(?=(\d\d\d)+(?!\d))/g, "$1.")
```

Comment:
Let's avoid adding fractions? Otherwise, it's hard to see what the actual number is:
![image](https://github.com/tuist/website/assets/9371695/3623e1a7-21ae-484e-ab02-a2fd5e2534b6)

Reply:
Luckily there's a package in NPM for everything :)

<img width="582" alt="image" src="https://github.com/tuist/website/assets/663605/4faf2b12-bdca-4bc4-a7b6-e24d2ac1fe71">